dev/source/docs/setting-up-sitl-on-macosx.rst

```diff
@@ -0,0 +1,137 @@
+.. _setting-up-sitl-on-macosx:
+
+===========================
+Setting up SITL on Mac OS X
+===========================
+
+This page describes how to setup the :ref:`SITL (Software In The Loop) <sitl-simulator-software-in-the-loop>` on Mac OS X. The specific commands were tested on macOS Catalina 10.15.7 and macOS Big Sur 11.1.
+
+Overview
+========
+
+The SITL simulator allows you to run Plane, Copter or Rover without any
+hardware. It is a build of the autopilot code using an ordinary C++
+compiler, giving you a native executable that allows you to test the
+behaviour of the code without hardware.
+
+SITL runs natively on Linux, Windows and Mac OSX.
+
+.. image:: ../images/SITL_macosx.png
+    :target: ../_images/SITL_macosx.png
+
+
+Install steps
+=============
+
+This guide assumes that a build environment is already setup for Ardupilot. If that is not the case please follow the available guides to setup the build environment for Mac OSX. The following step provide a condensed versions of the steps to take.
+
+
+Prerequisites
+-------------
+
+The following tools need to be installed to complete the installation. Between brackets the version used by the author at time of writing.
+
+ * Xcode (12.4)
+ * Xcode commandline tools (12.4)
+ * Homebrew (3.0.4-52-gbc37074)
+
+Preparing Python
+----------------
+
+Install the required python 3 version using homebrew and activate it. If you are familiar with pyenv or virtualenv it might be better to use one of those to setup the environment. This limits potential problems with other tools that require specific versions of python. How to do that is beyond the scope of this document.
+
+The following steps install python version 3.9.x for system wide usage.
+
+::
+
+	brew install python@3.9
+  export PATH="$(brew --prefix)/opt/python/libexec/bin":$PATH
+	
+	pip install wheel
+	pip install future pymavlink mavproxy
+
+Optional: Add the following to the either .bashrc or .zshrc to enable this version of python be default
+
+::
+
+  export PATH="$(brew --prefix)/opt/python/libexec/bin":$PATH
+
+
```

Comment:
It's also useful to add 

```bash
################################################################################
# ARDUPILOT
#
export PATH=$PATH:$HOME/Code/ardupilot/ardupilot/Tools/autotest
export ARDUPILOT_ROOT=$HOME/Code/ardupilot/ardupilot
```

to the shell initialisation scripts while you're at it. I put customisations including the PATH modification for Python in `~/.zprofile`.

Reply:
This mayncause problems if you run more than one clone in different directories. I often do that.